python/templates/endpoint_module.py.jinja

```diff
@@ -122,10 +147,21 @@ async def asyncio_detailed(
         {{ kwargs(endpoint) }}
     )
 
+    kwargs['headers'] = {'svix-req-id':f"{random.getrandbits(32)}", **kwargs['headers']}
```

Comment:
@tasn does random.getrandbits share a common seed? if so we should consider using `secrets.randbits`.  I know we'd end up with many clashes if we used math.rand in go vs crypt.rand.  Not sure if that's a concern here or not.

Reply:
It doesn't really matter that much, we just want something. Worst case it's not useful.